### PR TITLE
ERM-422 Internal Contact Role Filter

### PR DESCRIPTION
**Pull request by CalamityC**
add internal contact role filter

### CODE REVIEW

**Comment by CalamityC** [comment]
Could you just review if I implemented the filter correctly? Test will follow tomorrow

**Comment by doytch** [comment]
> Could you just review if I implemented the filter correctly? Test will follow tomorrow

Looks good!

File: translations/ui-agreements/en.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
   "agreements.agreementStatus": "Status",
   "agreements.isPerpetual": "Is perpetual",
   "agreements.internalContacts": "Internal contacts",
+  "agreements.internalContactsRole": "Internal contacts role",
```
Comment (doytch) [inline translations/ui-agreements/en.json]: I think `Internal contact role` (singular) would be better and would also match the verbiage used for `Organization role`

File: translations/ui-agreements/en.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
   "agreements.agreementStatus": "Status",
   "agreements.isPerpetual": "Is perpetual",
   "agreements.internalContacts": "Internal contacts",
-  "agreements.internalContactsRole": "Internal contacts role",
+  "agreements.internalContactsRole": "Internal contact   role",
```
Comment (doytch) [inline translations/ui-agreements/en.json]: Now there's an extra empty space. 😆

**Pull request merged**